src/Twemoji/index.js

```diff
@@ -25,6 +26,7 @@ export default class Twemoji extends React.Component {
 
   render() {
     const { children, ...other } = this.props;
+    delete other.options;
     return <div {...other}>{children}</div>;
```

Comment:
@raphaelhovsepyan 10a65788d456ddc452698ce23f1eb299766cf0c8 is good to go, 5ecfac0c5f01b2adf39811b1656e05c0a9d8e4ae has lint errors (run `npm run lint` to see)